docs/reference/autoscaling/deciders/proactive-storage-decider.asciidoc

```diff
@@ -0,0 +1,54 @@
+[role="xpack"]
+[testenv="enterprise"]
+[[autoscaling-proactive-storage-decider]]
+=== Proactive storage decider
+
+The proactive storage decider (`proactive_storage`) calculates the required storage to contain
+the current data set plus an estimated amount of expected additional data.
+
+It is enabled for all policies governing nodes with the `data_hot` role. A
+single configuration option `forecast_window` is available, defaulting to 30
+minutes.
+
+The estimation of expected additional data is based on past indexing that
+occurred within the `forecast_window`, i.e., by default the last 30 minutes.
+Only indexing into data streams contributes to the estimate.
```

Comment:
We'd typically put this in a `Parameters` or `Options` section with a def for `forecast_window`.

Reply:
Did this in d0429bd4c243159e1571238588e123dde1eeedf4